internal/grader/reject.go

```diff
@@ -45,10 +45,48 @@ func (this *RejectWindowMax) fullString(now timestamp.Timestamp) string {
 		nextTime.SafeMessage(), deltaString)
 }
 
-func checkForRejection(assignment *model.Assignment, submissionPath string, user string, message string) (RejectReason, error) {
+type RejectLateWithoutAllow struct {
+	AssignmentName string
+	DueDate        timestamp.Timestamp
+}
+
+func (this *RejectLateWithoutAllow) String() string {
+	return this.fullString(timestamp.Now())
+}
+
+func (this *RejectLateWithoutAllow) fullString(now timestamp.Timestamp) string {
+	deltaMS := now.ToMSecs() - this.DueDate.ToMSecs()
+	deltaString := time.Duration(deltaMS * int64(time.Millisecond)).String()
+
+	return fmt.Sprintf("Attempting to submit assignment (%s) late without 'allow late'."+
+		" It was due on %s (which was %s ago). Include the 'allow late' flag to submit an assignment late."+
```

Comment:
s/Include/Use/g

This is more general.
(Include isn't used for things like CLI.)